core/src/main/scala/bekindrewind/RecordOptions.scala

```diff
@@ -16,7 +18,19 @@ final case class RecordOptions(
 }
 
 object RecordOptions {
-  val default: RecordOptions = RecordOptions(_ => true, notRecordedThrowsErrors = false, overwriteAll = false)
+  val default: RecordOptions = RecordOptions(
+    _ => true,
+    notRecordedThrowsErrors = false,
+    overwriteAll = false,
+    requestTransformer = RequestTransformer.noop,
+    responseTransformer = ResponseTransformer.noop
```

Comment:
It's a stylistic choice, but I'm okay with just using `identity` and getting rid of these `noop` aliases.